### ISSUE TITLE
can't commit merge with conflicts, when resolving the conflicts results in no changes

### ISSUE THREAD

**Issue by adam-ce** [JetpackDuba/Gitnuro#120]
**Describe the bug**
I pulled and had to merge due to conflicts. After merging with git mergetool (from command line), it happened, that no additional files needed to be staged (because merging resulted in a choosing the local version everywhere). it was not possible to complete the merge in nuro.

**To Reproduce**

1. Create a merge conflict
2. select local version everywhere
3. try to commit. the merge/commit button is greyed out

**Expected behavior**
the merge button is active, and completes the merge

**Desktop (please complete the following information):**
- OS: Kubuntu 22.04
- Version 1.2.1 (from flathub)

**Comment by adam-ce** [comment]
no. that's the point. there are no unstaged changes. so the button is disabled. therefore, i can't finish the merge.

**Comment by JetpackDuba** [comment]
I've noticed an issue where the button is enabled but it fails saying "No changes", which should be fixed now in commit this commit https://github.com/JetpackDuba/Gitnuro/commit/b2dc0ecc50af6facdc65329c5c30ca55088dbe61 but I can't reproduce the button being disabled.

Steps followed:

Same file changed in both branches (local & remote):
![imagen](https://github.com/JetpackDuba/Gitnuro/assets/16917468/ab45bdc0-9b84-47b0-a0f5-33680845cf05)

A conflict shows up when pulling:
![imagen](https://github.com/JetpackDuba/Gitnuro/assets/16917468/792c2012-b9cb-4e26-ad76-e6959bc16856)

Conflict is fixed (button  is enabled):
![imagen](https://github.com/JetpackDuba/Gitnuro/assets/16917468/77e9d227-62ee-4b4a-b9ce-4263e51a42b4)

At this point, before the previously mentioned commit, it would show an error popup but with that change it works as expected by creating an empty commit:
![imagen](https://github.com/JetpackDuba/Gitnuro/assets/16917468/d9034d16-aef5-4e3a-b60e-5354584950ed)


Am I missing something?

**Comment by adam-ce** [comment]
how did you resolve the conflict? i used git mergetool.

**Comment by adam-ce** [comment]
ah, my merge tool (kdiff3) keeps a file_with_conflict.cpp.orig file. after deleting it i the merge button gets enabled.
![image](https://github.com/JetpackDuba/Gitnuro/assets/5292991/9a0739c8-366d-4ff9-b6c8-2450d37e724b)

i guess this could be a feature to keep the repository clean, but imo it's not intuitive. at least there should be an information why the button is disabled :)

**Comment by JetpackDuba** [comment]
That makes sense.  I may add a setting to change this behavior, but not sure yet.

I've edited the title and your first message to reflect properly the issue.
